### PR TITLE
Add http status processor doc

### PR DESCRIPTION
**Pull request by Mrod1598**
##### Checklist
- [ ] Changes are tested
- [ ] CI has passed

### CODE REVIEW

File: docs/www/integrations/processors/filter_http_status.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+title: Filter HTTP Status
+category: 636c08e1212e49001e7a3032
+parentDoc: 636c0a0cddea2a005d14423a
+slug: filter-severity
+hidden: false
+---
+
+##  HTTP Status Filter Processor
+
+The Severity Filter processor can be used to filter out logs that do not meet a given severity threshold.
```
Comment (dehaansa) [inline docs/www/integrations/processors/filter_http_status.md:11]: this is wrong copy?

**Review by dehaansa — CHANGES_REQUESTED** [review]

**Review by dehaansa — APPROVED** [review]

File: docs/www/integrations/processors/filter_http_status.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
+---
+title: Filter HTTP Status
+category: 636c08e1212e49001e7a3032
+parentDoc: 636c0a0cddea2a005d14423a
+slug: filter-severity
```
Comment (antonblock) [inline docs/www/integrations/processors/filter_http_status.md:5]:
```suggestion
slug: filter-http-status
```

File: docs/www/integrations/processors/filter_http_status.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+hidden: false
+---
+
+##  HTTP Status Filter Processor
+
+filter out logs that contain a status code between a min and max
```
Comment (antonblock) [inline docs/www/integrations/processors/filter_http_status.md:11]:
```suggestion
Filter out logs containing a status code between a min and max.
```

**Review by dehaansa — APPROVED** [review]

File: docs/www/integrations/processors/filter_http_status.md (hunk 2 above)
Comment (Mrod1598) [reply docs/www/integrations/processors/filter_http_status.md]: updated.

**Review by antonblock — APPROVED** [review]

**Pull request merged**